examples/flink/pom.xml

```diff
@@ -74,6 +72,12 @@
       <version>${project.version}</version>
     </dependency>
 
+    <dependency>
```

Comment:
@merlimat 
Sorry, this dependency is not needed as its inherited from the root pom.
I will create a PR fixing this asap

Reply:
Ok, the buildtools is anyway injected as a dependency for all the modules

Reply:
Yes, my bad

Reply:
fixed by #7560